### DISCUSSION TITLE
container

### DISCUSSION THREAD

**Discussion by sobhardwaj** [mantinedev/mantine#55] [Q&A] [answered]
I just started today, and  pick up main container component, not have enough property there to make the contianer smoonth? 

<Container style={{ backgroundColor: '#12bc86', height: 50, borderRadius : '10px 10px 0px 0px' }}>
            </Container>

<Container   shadow="md" radius="md" </Container>

**Comment by rtivital** [comment, marked as answer]
Container is a layout component, use it to center things with given width, to add shadows and radius use Paper – https://mantine.dev/core/paper/
